ui/app/AppLayouts/Wallet/popups/swap/SwapModal.qml

```diff
@@ -97,6 +102,36 @@ StatusDialog {
         StatusBaseText {
             text: qsTr("Selected to token: %1").arg(swapInputParamsForm.toTokenKey)
         }
+        StatusButton {
```

Comment:
I dont think you'll keep this for your final PR right? or we want to keep it this way?

Reply:
for sure! 
The textbox and the first buttons should be removed once we properly get/show the swap proposal (https://github.com/status-im/status-desktop/issues/14828)
The other buttons should be removed with https://github.com/status-im/status-desktop/issues/14833 and https://github.com/status-im/status-desktop/issues/14835

Reply:
or you mean remove them when I de-draft this one? I think it's useful to have the placeholders until we integrate the real UI components

Reply:
yes I meant not adding any qml changes in your PR I working on swap proposal right now. but if you think it is useful still, we can keep it, no problem.